ui/src/app/store/host/types.ts

```diff
@@ -0,0 +1,4 @@
+import type { Controller } from "app/store/controller/types";
```

Comment:
If our goal with this refactor is to have every subdirectory under `store` a representation of a slice of state, then I don't think `store/host` makes sense. I guess this is the problem with having more granular type modules. Perhaps just co-locate this with Machine?

Reply:
Moved to store/types/host